### PR TITLE
feat:Adding Builders Journey

### PR DESCRIPTION
**Pull request by albrich-me**
**Issue #:**

**Description of changes:** Adding the Builders Journey document

**Testing**

1. How did you test these changes?
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
3. If applicable, have you run `npm run build` successfully locally to fix all warnings and errors?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by schuettc — APPROVED** [review]
Documentation only

File: BUILDERS_JOURNEY.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
+Version 6
```
Comment (ltrung) [inline BUILDERS_JOURNEY.md:1]: What does this version mean?

**Comment by albrich-me** [comment]
It was my attempt to track the version without changing the file name.  I can remove it.

Rick Albertson
Sr. Chime Specialist Solutions Architect
Amazon Web Services
E: ***@***.***
M: +1-540-819-0630
 [signature_1933226851]


From: Trung Le ***@***.***>
Reply-To: aws-samples/amazon-chime-sdk ***@***.***>
Date: Monday, August 15, 2022 at 5:43 PM
To: aws-samples/amazon-chime-sdk ***@***.***>
Cc: "Albertson, Rick" ***@***.***>, Author ***@***.***>
Subject: Re: [aws-samples/amazon-chime-sdk] feat:Adding Builders Journey (PR #222)


@ltrung commented on this pull request.

________________________________

In BUILDERS_JOURNEY.md<https://github.com/aws-samples/amazon-chime-sdk/pull/222#discussion_r946153148>:

> @@ -0,0 +1,501 @@

+Version 6

What does this version mean?

**Comment by albrich-me** [comment]
I resubmitted it under amazon-chime-sdk-js/guides/README.md without the Version line at the beginning.

Regards,

Rick Albertson
Sr. Chime Specialist Solutions Architect
Amazon Web Services
E: ***@***.***
M: +1-540-819-0630
 [signature_1933226851]


From: Trung Le ***@***.***>
Reply-To: aws-samples/amazon-chime-sdk ***@***.***>
Date: Monday, August 15, 2022 at 5:43 PM
To: aws-samples/amazon-chime-sdk ***@***.***>
Cc: "Albertson, Rick" ***@***.***>, Author ***@***.***>
Subject: Re: [aws-samples/amazon-chime-sdk] feat:Adding Builders Journey (PR #222)


@ltrung commented on this pull request.

________________________________

In BUILDERS_JOURNEY.md<https://github.com/aws-samples/amazon-chime-sdk/pull/222#discussion_r946153148>:

> @@ -0,0 +1,501 @@

+Version 6

What does this version mean?

**Pull request closed without merging**